string chip_id
string serial_number
string core_part_number